_data-prepper/pipelines/configuration/processors/mutate-event.md

```diff
@@ -0,0 +1,302 @@
+
+```json
+{"message": "value"}
+```
+
+When you run the `AddEntries` processor, it parses the message into the following output:
+
+```json
+{"message": "value", "newMessage": 3}
+```
+
+> If `newMessage` already exists, its existing value is overwritten with a value of `3`.
+
+In the preceding example, the `add_entries` processor adds a new entry `{"newMessage": 3}` to the existing event, `{"message": "value"}` so that the new event contains two entries in the final output, which iss `{"message": "value","newMessage": 3}`.
+
+## copy value
+
+The `copy value` processor copies the values of an existing key within an event to another key. For example, in the following Usage section, the `copy value` processor copies the value of the key "message", which is "value" to another key called "newMessage". The value of the key "newMessage" is also "value".
+
+<!-- Is there a way to link to this specific Usage section and not another one? They all have the same title. Should they be renamed to match their respective processors?--->
+
+### Configuration
+
+You can configure the `copy value` processor with the following options.
+
+Option | Required | Description 
+:--- | :--- | :---
+| `entries` | Yes | A list of entries to be copied in an event. |
+| `from_key` | Yes | The key of the entry to be copied.
+| `to_key` | Yes | The key of the new entry to be added.
+| `overwrite_if_to_key_exists` | No | When set to a value of `true`, if `to_key` already exists in the event, then the existing value will be overwritten. The default value is `false`. |
+
+### Usage
+
+To get started, create the following `pipeline.yaml` file:
+
+```yaml
+pipeline:
+  source:
+    file:
+      path: "/full/path/to/logs_json.log"
+      record_type: "event"
+      format: "json"
+  processor:
+    - copy_values:
+        entries:
+        - from_key: "message"
+          to_key: "newMessage"
+          overwrite_if_to_key_exists: true
+  sink:
+    - stdout:
+```
+
+Next, create a log file named `logs_json.log` and replace the `path` in the file source of your `pipeline.yaml` with this filepath. For more information, see [Configuring Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/getting-started/#2-configuring-data-prepper).
+
+```json
+{"message": "value"}
+```
+
+When you run this processor, it parses the message into the following output:
+
+```json
+{"message": "value", "newMessage": "value"}
+```
+
+> If `newMessage` had already exists, its current value is overwritten with `value`.
+
+
+## DeleteEntry
+
+The `DeleteEntry` processor deletes entries in an event, such as key-value pairs. You can define the keys you want to delete in the `with-keys` field following `delete_entries` in the YAML configuration file. Those keys along with their values are deleted. See the example below in the Usage section, where `with_keys: ["message"] exists in the YAML file. This means that the entry {"message": "value"} is deleted. 
```

Comment:
Again, about referring to the example two headings down, it becomes kind of redundant to show it again once you finally get there. Might want to let the Usage heading do the work.